libgalois/src/analytics/random_walks/random_walks.cpp

```diff
@@ -485,6 +481,12 @@ RandomWalksWithWrap(katana::PropertyGraph* pg, RandomWalksPlan plan) {
     return res.error();
   }
 
+  // TODO(amp): This is incorrect. For Node2vec this needs to be:
+  //    Algorithm::Graph::Make(pg, {}, {}) // Ignoring all properties.
+  //  For Edge2vec this needs to be:
+  //    Algorithm::Graph::Make(pg, {}, {edge_type_property_name})
+  //  The current version requires the input to have exactly the properties
+  //  expected by the algorithm implementation.
```

Comment:
This issue needs to be addressed to wrap `RandomWalks`. The algorithm will currently not work.

Reply:
Good point. We need to separate them out into different applications.